CHANGELOG.md

```diff
@@ -4,460 +4,460 @@ All notable changes to this project will be documented in this file. Dates are d
 
 Generated by [`auto-changelog`](https://github.com/CookPete/auto-changelog).
 
-#### [v9.12.0](https://github.com/contiamo/restful-react/compare/v9.11.1...v9.12.0)
+#### [v9.12.1](https://github.com/mbrookson/restful-react/compare/v9.11.1...v9.12.1)
```

Comment:
we missed this one

Reply:
Oops! That's a side-effect of running `npm version patch` on my local machine. 😱

Reply:
Not a big issue ^^